src/pint/models/timing_model.py

```diff
@@ -2265,7 +2296,7 @@ def get_prefix_mapping(self, prefix):
             par = getattr(self, parname)
             if par.is_prefix and par.prefix == prefix:
                 mapping[par.index] = parname
-        return mapping
+        return OrderedDict(sorted(mapping.items()))
```

Comment:
Why an OrderedDict?

Reply:
This is to make sure the map in the prefix mapping is ordered by the index.